src/jit/lclvars.cpp

```diff
@@ -3363,6 +3363,24 @@ void Compiler::lvaAllocOutgoingArgSpace()
         lvaTable[lvaOutgoingArgSpaceVar].lvRefCnt     = 1;
         lvaTable[lvaOutgoingArgSpaceVar].lvRefCntWtd  = BB_UNITY_WEIGHT;
 
+        // For debug IL code compilation we do not create IR for calls to THROW_HELPERs.
```

Comment:
I think that this sequence would be more clear if we started a if then block here using:

if (isWindowsAmd64ABI())
{ 
    // The Windows ABI for Amd64 requires us to reserve 4 slots